lua/nvim-tree/explorer/sorters.lua

```diff
@@ -73,6 +73,9 @@ function M.merge_sort(t, comparator)
   end
 
   split_merge(t, 1, #t, comparator)
+  if M.after_sort then
```

Comment:
Do we need to run the comparator and split_merge first?

We should call the user's after_sort instead of M.merge_sort.

Reply:
OMZ,,, I miss understood..
I thought 'sort_by' comparator must be run through merge sort function

Reply:
> OMZ,,, I miss understood.. I thought 'sort_by' comparator must be run through merge sort function

We would need to try it.

Please try `explorer.lua` and `reload.lua` calling `after_sort` directly.

Reply:
i will try this too☺️